### DISCUSSION TITLE
云审查这个插件怎么使用

### DISCUSSION THREAD

**Discussion by mnwgkj** [MoeNetwork/Tieba-Cloud-Sign#204] [疑问] [answered]
![image](https://user-images.githubusercontent.com/109659250/189476671-7a0e98ca-75b4-4190-b350-5e22fb6fea1a.png)
这样设置半小时后还是没有日志

**Comment by kenvix** [comment]
Issue区不是论坛，只用于提出程序错误

**Comment by mnwgkj** [comment]
有谈论的群组之类吗

**Reply by BANKA2017** [reply]
就Discussions这里即可

**Comment by BANKA2017** [comment, marked as answer]
云审查已经很久没维护了，加上最近贴吧接口大改，可能已经失效

**Reply by mnwgkj** [reply]
好吧 吧务没啥方便的插件了……

**Reply by mnwgkj** [reply]
失效的插件在检查更新的时候应该自动去除 不写入 不然都不知道哪个能用 挨个来问

**Reply by n0099** [reply]
是之前 #193 引入的

**Reply by n0099** [reply]
> 好吧 吧务没啥方便的插件了……

建议使用[dog194版贴吧管理器](https://github.com/dog194/TiebaManager)，[starry版贴吧管理器](https://github.com/Starry-OvO/Tieba-Manager)，鸡血神贴吧辅助工具

**Reply by mnwgkj** [reply]
电脑版本的  不能部署服务器上

**Reply by n0099** [reply]
> 电脑版本的 不能部署服务器上

那就用第二个py写的
或者淘宝搜索挂机宝买个垃圾win服来跑管理器
